extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -18,10 +18,85 @@ amp-story-dev-tools {
   width: 100% !important;
   height: 100% !important;
   box-sizing: border-box !important;
-  background-color: #121212 !important;
+  background-color: black !important;
   font-size: 16px !important;
   font-family: 'Poppins', sans-serif !important;
   color: white !important;
   padding: 30px 10% !important;
   text-align: center !important;
 }
+
+.i-amphtml-story-dev-tools-container {
+  height: 100% !important;
+}
+
+.i-amphtml-story-dev-tools-header {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+  color: white !important;
+  height: 40px !important;
+  box-sizing: border-box !important;
+}
+
+.i-amphtml-story-dev-tools-brand {
+  display: inline-flex !important;
+}
+
+.i-amphtml-story-dev-tools-brand-text > span {
+  display: block !important;
+  letter-spacing: 0.1em !important;
+  line-height: 1.2em !important;
+  padding-left: 10px !important;
+}
+
+.i-amphtml-story-dev-tools-brand-text > span:nth-child(2) {
+  color: #79B3FF !important;
+  font-size: 10px !important;
+  text-align: left;
+}
+
+.i-amphtml-story-dev-tools-button {
+  cursor: pointer !important;
+  font-size: 13px !important;
+  display: flex !important;
+  align-items: center !important;
+  border: 1px solid #79B3FF55 !important;
+  border-radius: 20px !important;
+  padding: 0 6px 0 10px !important;
+  height: 32px !important;
+  box-sizing: border-box !important;
+  color: #79B3FF !important;
+  transition: background 0.05s ease-in-out !important;
```

Comment:
Super fast transition. I don't think it's visible. 

I wonder if we want to leave this out for now and do animation work when we can see more of the elements coming together.

Reply:
Makes sense to leave this for later